src/modules/tabs/tab-button.component.scss

```diff
@@ -16,3 +16,8 @@
     color: #fff;
   }
 }
+
+.sky-tab-header-count {
```

Comment:
We need to look at having these counts blue when the tab isn't selected. See the docs for Sky 1